doc/source/user/building.rst

```diff
@@ -35,6 +35,9 @@ Building NumPy requires the following software installed:
    installed. However, if these are available, NumPy's setup script can detect
    them and use them for building. A number of different LAPACK library setups
    can be used, including optimized LAPACK libraries such as OpenBLAS or MKL.
+   A ``site.cfg`` file, if found in the base directory of the sources, can be
+   used to specify locations of the libraries and headers, and the preferences
+   between them. See the sample ``site.cfg.example`` for more information. 
```

Comment:
"Between" is the wrong word here. See above for a general style suggestion. I think you can omit the "... and the... then" phrase,

Reply:
fixed